BotwSaveManager/Resource.cs

```diff
@@ -0,0 +1,57 @@
+﻿using BotwSaveManager.Core.Helpers;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net.Http;
+using System.Reflection;
+using System.Text;
+using System.Text.Json;
+
+namespace BotwSaveManager
+{
+    public class Resource
+    {
+        internal static readonly string Namespace = typeof(Resource).Namespace ?? "BotwSaveManager";
+        internal static readonly string BaseUrl = $"https://raw.githubusercontent.com/ArchLeaders/{Namespace}/master";
+
+        public byte[] Data { get; set; }
+        public Resource(byte[] data) => Data = data;
+        public Resource(string name, string root)
+        {
+            if (File.Exists($"{root}/{name}")) {
```

Comment:
When combining paths in cross-platform applications, it is better to use `Path.Combine` then to concatenate two paths, as there are a few (uncommon) cases where putting a slash between two paths causes issues.